plugins/inventory/icinga2.py

```diff
@@ -35,13 +35,22 @@
         type: string
         required: true
       host_filter:
-        description: An Icinga2 API valid host filter.
+        description:
+          - An Icinga2 API valid host filter. Leave blank for no filtering
+          - Note: Quotations must be escaped
```

Comment:
```suggestion
```
That is a YAML formatting issue. Explaining YAML in module options is something we should not do.